evap/results/templates/results_course_detail.html

```diff
@@ -35,9 +35,14 @@ <h3 class="mr-auto">{{ course.name }} ({{ course.semester.name }})</h3>
             <div class="btn-switch my-auto d-print-none">
                 <div class="btn-switch-label">{% trans 'View as' %}</div>
                 <div class="btn-group">
-                    <a href="{% url 'results:course_detail' course.semester.id course.id %}?public_view=false" role="button" class="btn btn-sm btn-light{% if not public_view %} active{% endif %}" data-toggle="tooltip" data-placement="top"
+                    {% if user.is_staff and view == 'filter' or contributor %}
+                        <a href="{% url 'results:course_detail' course.semester.id course.id %}?view=filter" role="button" class="btn btn-sm btn-light{% if view == 'filter' %} active{% endif %}" data-toggle="tooltip" data-placement="top"
+                        title="{% trans 'Shows filtered view for personal export.' %}">
```

Comment:
please make this more verbose, like `Shows filtered view meant for personal export. Only your personal results and those about the course. Other contributors' results and private answers are hidden.`